unity migration (#3668)
getsentry/sentry-docs: src/platforms/unity/migration.mdx

---
title: Migration Guide
sidebar_order: 1000
redirect_from:
  - /platforms/unity/unity-lite/
---

## Migrating from `sentry.unity.lite` to `sentry.unity`.

> <Note>
> 
> The new Sentry Unity SDK requires Unity version 2018.1 or higher. With .NET Standard 2.0 scripting profile.
> Event submission happen on a background thread which isn't available on WebGL players at this point.
> 
> </Note>
> 
> Sentry is deprecating the Sentry Unity Lite SDK for Mobile, Desktop and Console players.
> Customers using Unity 5.x or WebGL can still use the Sentry Unity Lite as it continues to be compatible with sentry.io.

Reviewer: ```suggestion
<Note>

The Sentry Unity Lite SDK is deprecated for mobile, desktop, and console players. Customers using Unity 5.x or WebGL can still use the Sentry Unity Lite as it continues to be compatible with sentry.io.

The updated Sentry Unity SDK requires Unity version 2018.1 or higher with .NET Standard 2.0 scripting profile.
Event submission happen on a background thread, which isn't currently available on WebGL players.

</Note>
```

> ### To migrate, follow the following steps:

Reviewer: delete this heading

1. Remove the old `Sentry.cs` and `SentrySdk.cs` files from your project.
2. Remove the old initialization code `gameObject.AddComponent<SentrySdk>().Dsn = "___PUBLIC_DSN___";`.
3. Install the new [Sentry Unity SDK](/platforms/unity/).

> 4. Calls to the API such as `SentrySdk.CaptureMessage("Test event");` will continue to work and don't require any change.
> 5. Browse the documentation to learn more about the new capabilities of the SDK such as [enriching events](enriching-events/), [performance monitoring](performance/) and [configuration](configuration/).

Reviewer: These aren't steps :) 
```suggestion

Calls to the API, such as `SentrySdk.CaptureMessage("Test event");` will continue to work and don't require any change. Browse the documentation to learn more about the new capabilities of the SDK such as [enriching events](enriching-events/), [performance monitoring](performance/) and [configuration](configuration/).
```
